### PR TITLE
html-skeleton - Matt Cha

### PR DESCRIPTION
**Pull request by matt-cha**
Screen Shot 2022-12-05 at 10 58 49 AM

### CODE REVIEW

File: html-skeleton/index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE>
+<html>
+  <head>
+    <title>HTML Skeletons</title>
+    <body>Hello, HTML!</body>
```
Comment (codingcodymiller) [inline html-skeleton/index.html:5]: `body` needs to be a sibling of `head`, not a child of it

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-skeleton/index.html (hunk 1 above)
Comment (matt-cha) [reply html-skeleton/index.html]: Screen Shot 2022-12-05 at 11 30 08 AM
Updated the change thank you!

File: html-skeleton/index.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE>
```
Comment (codingcodymiller) [inline html-skeleton/index.html:1]: This needs to be `<!DOCTYPE html>`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-skeleton/index.html (hunk 2 above)
Comment (matt-cha) [reply html-skeleton/index.html]: Screen Shot 2022-12-05 at 12 12 29 PM
Added this as well thank you!

File: html-skeleton/index.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE HTML>
```
Comment (codingcodymiller) [inline html-skeleton/index.html:1]: `html` needs to be lowercase here 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-skeleton/index.html (hunk 3 above)
Comment (matt-cha) [reply html-skeleton/index.html]: Screen Shot 2022-12-05 at 2 07 40 PM
Fixed this thanks Cody!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**